solc/jsonCompiler.cpp

```diff
@@ -305,6 +307,11 @@ static string s_outputBuffer;
 
 extern "C"
 {
+extern char const* license()
+{
+	/// TOOD: include the copyright information on the top.
```

Comment:
It is not a todo for this PR as we haven't decided on the format of the copyright.

The the top of the license can include the copyright.